spec/jobs/bag_move_job_spec.rb

```diff
@@ -104,31 +117,29 @@ class InjectedError < RuntimeError
     end
 
     context "when validation raises an exception" do
-      let(:validator) { double(:validator) }
-
-      subject(:run_job) { described_class.perform_now(queue_item, validator: validator) }
+      subject(:run_job) { described_class.perform_now(queue_item) }
 
       before(:each) do
-        allow(validator).to receive(:valid?).and_raise InjectedError, "injected error"
+        allow(queue_item.package).to receive(:valid_for_ingest?).and_raise("arbitrary failure")
       end
 
       it "re-raises the exception" do
-        expect { run_job }.to raise_exception(InjectedError)
+        expect { run_job }.to raise_exception(/arbitrary failure/)
```

Comment:
My initial reaction was that InjectedError was there for a reason, but I think this is still fine, as it's testing that it raises the exception we put in there rather than some arbitrary other exception.

Reply:
Thanks. I've converted these to just use StandardError and match an example-oriented message in each case.